packages/kcmsf/src/pages/ranking.tsx

```diff
@@ -33,6 +34,11 @@ export const Ranking = () => {
   >(new Map());
   const navigate = useNavigate();
 
+  const [delay, setDelay] = useState(10000);
+  const [isAutoReload, setIsAutoReload] = useState(false);
+
+  const [fetchTime, setFetchTime] = useState<string>("00:00");
```

Comment:
型で縛ってもいいかもしれない(必須ではない)
```suggestion
  const [fetchTime, setFetchTime] = useState<`${string}:${string}`>("00:00");
```